CriticalMass/URLCodable.swift

```diff
@@ -46,28 +46,34 @@ extension URLCodable {
         return try self.init(scheme: urlComponents.scheme, host: urlComponents.host, path: urlComponents.path, queryObject: codableObject)
     }
 
-    private func encode<T: Encodable>(codable: T) throws -> [URLQueryItem] {
+    private static func decode<T: Codable>(type: T.Type, from items: [URLQueryItem]) throws -> T {
```

Comment:
Maybe the function should be renamed since it does a lot more than just to decode

Reply:
In my opinion the name makes sense because it's a known pattern from the Codable protocol. Example
```swift
let followURLObject = try FollowURLObject.decode(from: url.absoluteString)
```

But I'm open for suggestions